### PR TITLE
chore(internal): additional tests

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
Additional tests in support of https://github.com/NullVoxPopuli/eslint-plugin-decorator-position/issues/474

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
@k-teplyakov any chance you can cherry-pick these changes to a new PR targeting `main`?

This PR has gotten a bit stale, and there are a good number of conflicts :sweat_smile:

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -46,6 +46,20 @@ jobs:
     - run: yarn install
     - run: yarn lint:docs-js
 
+  tooling:
```
Comment (NullVoxPopuli) [inline .github/workflows/lint.yml:49]: we don't need this :upside_down_face:

Comment (k-teplyakov) [reply .github/workflows/lint.yml:49]: > @k-teplyakov any chance you can cherry-pick these changes to a new PR targeting `main`?
> 
> This PR has gotten a bit stale, and there are a good number of conflicts 😅

sure. i will make new one

Comment (NullVoxPopuli) [reply .github/workflows/lint.yml:49]: thank you !! <3

**Comment by NullVoxPopuli** [comment]
Closing in favor of @k-teplyakov 's work

**Pull request closed without merging**